test/accessibility.test.js

```diff
@@ -2,44 +2,164 @@ import { assertEquals } from 'https://deno.land/std@0.128.0/testing/asserts.ts';
 import { wordWrap } from '../src/accessibility.js';
 
 Deno.test('wordWrap splits a string into an array of lines ', () => {
-  const input =
-    'Sometimes science is more art than science, Morty. A lot of people dont get that';
+  const input = {
+    text:
+      'Sometimes science is more art than science, Morty. A lot of people dont get that',
+    wordWrap: 30,
+  };
   const expected = [
-    'Sometimes science is more art',
-    'than science, Morty. A lot of',
-    'people dont get that',
+    {
+      text: 'Sometimes science is more art',
+    },
+    {
+      text: 'than science, Morty. A lot of',
+    },
+    {
+      text: 'people dont get that',
+    },
   ];
-  const actual = wordWrap(input, 30);
+  //const actual = wordWrap(input);
+  assertEquals(wordWrap(input), expected);
+});
+
+Deno.test('wordWrap keeps object feilds other than wordWrap constant ', () => {
+  const input = {
+    text:
+      'Sometimes science is more art than science, Morty. A lot of people dont get that',
+    wordWrap: 30,
+    color: 'red',
+    size: 12,
+  };
+  const expected = [
+    {
+      text: 'Sometimes science is more art',
+      color: 'red',
+      size: 12,
+    },
+    {
+      text: 'than science, Morty. A lot of',
+      color: 'red',
+      size: 12,
+    },
+    {
+      text: 'people dont get that',
+      color: 'red',
+      size: 12,
+    },
+  ];
+  const actual = wordWrap(input);
   assertEquals(actual, expected);
 });
 
 Deno.test('wordWrap places words longer than max line length on their own line', () => {
```

Comment:
maybe 
'wordWrap splits phrases longer than max line length in to new lines' ?

Reply:
The description I currently have does not reflect the intent of that test. The intent was to prove that long words aren't getting broken into two lines. I changed it to 'wordWrap does not split words longer than the max line length into multiple lines'.